test/bzmesh.jl

```diff
@@ -7,16 +7,16 @@ include("testcases.jl")
 
 
 @testset "bzmesh_uniform agrees with spglib" begin
-    spglib = pyimport_conda("spglib", "spglib")
 
     function test_against_spglib(kgrid_size; kshift=[0, 0, 0])
         kgrid_size = Vec3(kgrid_size)
-        identity = [reshape(Mat3{Int}(I), 1, 3, 3)]
+        identity = collect(reshape(Matrix{Cint}(I, 3, 3), 1, 3, 3))
         is_shift = ifelse.(kshift .== 0, 0, 1)
-        _, grid = spglib.get_stabilized_reciprocal_mesh(kgrid_size, identity,
+        num_kpts, _, grid = DFTK.spglib_get_stabilized_reciprocal_mesh(kgrid_size, identity,
```

Comment:
If you don't use `num_kpts` please also use an underscore for that arg.

Reply:
Actually I just had an idea about this: Basically what this tries to test is that we agree with spglib, so from that point of view the best way to test would be (contrary to what I said before) to directly do a ccall into spglib_jll from the test and bypass as much of the extra icing DFTK spglib.jl adds on top. You're call if you think that's too much effort to do or not ...

Reply:
Also the line starts to become a bit long ;)